src/converter.js

```diff
@@ -1,6 +1,11 @@
-import toNow from 'date-fns/distance_in_words_to_now'
+import toNow from 'date-fns/formatDistanceToNow'
+import parseISO from 'date-fns/parseISO'
 
 export default (date, locale, converterOptions) => {
+  if (typeof date === 'string') {
+    date = parseISO(date)
```

Comment:
why not simply `new Date()`

Reply:
`new Date`??? <- this method will generate now.
we need to convert string dates before passing to `date-fns`.

Reply:
https://blog.date-fns.org/post/we-cut-date-fns-v2-minimal-build-size-down-to-300-bytes-and-now-its-the-smallest-date-library-18f2nvh2z0yal/